client/components/layout/stylesheets/sidebar.styl

```diff
@@ -57,40 +59,40 @@
     img
       opacity 1
       filter none
-      -webkit-filter none
     #layout-sidebar-profile-menu
       display block
   img
     border-radius 100px
-    width 50px
+    width 40px
     display inline-block
     margin-bottom 10px
     filter grayscale(100%)
-    -webkit-filter grayscale(100%)
-    transition filter 0.3s
-    -webkit-transition -webkit-filter 0.3s ease
+    transition filter 0.3s ease-in-out, opacity 0.3s ease-in-out
+    opacity 0.8
   #layout-sidebar-profile-menu
     position absolute
-    width 180px
+    width 220px
     background black
-    left sidebar-width + 20
+    left sidebar-width
+    border-left 20px solid white
```

Comment:
CSS is all about the bag of ~~hacks~~ tricks you acquire over time :stuck_out_tongue_winking_eye:.